index.html

```diff
@@ -12592,6 +12832,97 @@ <h3>Serializing/Deserializing RDF</h3>
+    <p>Datatypes based on this namespace allow round-tripping of JSON-LD documents using base direction,
+      although the mechanism is not otherwise standardized.</p>
+    <p>The <a data-cite="JSON-LD11-API#deserialize-json-ld-to-rdf-algorithm">Deserialize JSON-LD to RDF Algorithm</a>
+      can be used with the <dfn data-cite="JSON-LD11-API#dom-jsonldoptions-rdfDirection">rdfDirection</dfn>
+      set to `datatype` to generate <a>RDF literals</a> using the `i18n` base to create an IRI
+      encoding the <a>base direction</a> along with optional <a>language tag</a> from
+      value objects containing `@direction` by appending to `https://www.w3.org/i18n#`
+      the value of `@language`, if any, followed by an underscore (`"_"`) followed
+      by the value of `@direction`.</p>
+
+    <p>The following example shows two statements with literal values of `i18n:ar_rtl`,
+      which encodes the language tag `ar` and the base direction `rtl`.</p>
+    <pre class="turtle-dt nohighlight"
+         data-content-type="text/turtle"
+         data-transform="updateExample"
+         data-options="rdfDirection=datatype"
+         data-to-rdf>
+    <!--
+    @prefix ex: <http://example.org/> .
+    @prefix i18n: <https://www.w3.org/i18n#> .
+
+    # Note that this version preserves the base direction using a datatype.
+    [
+      ex:title ****"HTML و CSS: تصميم و إنشاء مواقع الويب"^^i18n:ar_rtl****;
+      ex:publisher ****"مكتبة"^^i18n:ar_rtl****
+    ] .
+    -->
+    </pre>
+    <p>See <a href="#base-direction" class="sectionRef"></a> for more details
+      on using <a>base direction</a> for strings.</p>
+  </section>
+
+  <section class='informative'><h2>The `rdf:CompoundLiteral` class and the `rdf:language` and `rdf:direction` Datatypes</h2>
+    <p>This specification defines the `rdf:CompoundLiteral` datatype, which is in the domain
+      of `rdf:language` and `rdf:direction` to be used for describing RDF literal values
+      containing base direction and a possible language tag to be associated with the
+      string value of `rdf:value` on the same subject.</p>
+
+    <dl>
+      <dt>`rdf:CompoundLiteral`</dt>
+      <dd>A class representing a compound literal.</dd>
+      <dt>`rdf:language`</dt>
+      <dd>An RDF <a>property</a>.
+        The range of the property is an `rdfs:Literal`, whose value MUST be a well-formed [[BCP47]] <a>language tag</a>.
+        The domain of the property is `rdf:CompoundLiteral`.</dd>
+      <dt>`rdf:direction`</dt>
+      <dd>An RDF <a>property</a>.
+        The range of the property is an `rdfs:Literal`, whose value MUST be either `"ltr"` or `"rtl"`.
+        The domain of the property is `rdf:CompoundLiteral`.</dd>
+    </dl>
+
+    <p>The <a data-cite="JSON-LD11-API#deserialize-json-ld-to-rdf-algorithm">Deserialize JSON-LD to RDF Algorithm</a>
+      can be used with the <a>rdfDirection</a>
+      set to `bnode` to generate <a>RDF literals</a> using these properties to
```

Comment:
I would suggest something more specific, such as `compound-literal`, in case other implementation come up with alternative bnode-based solution